src/controllers.js

```diff
@@ -19,6 +19,17 @@ router.get("/div/:a/:b", async function (req, res) {
     }
 });
 
+router.get("/pow/:a", async function (req, res) {
+    try {
+        const a = Number(req.params.a);
+         if (isNaN(a))
```

Comment:
Should use the function getRequestParameters to get the parameters and check if they are valid